packages/jupyter-ai-magics/jupyter_ai_magics/parsers.py

```diff
@@ -32,6 +33,22 @@
     + "does nothing with other providers."
 )
 
+ENDPOINT_ARGS_SHORT_OPTION = "-e"
+ENDPOINT_ARGS_LONG_OPTION = "--endpoint-kwargs"
+ENDPOINT_ARGS_HELP = (
+    "A JSON value that specifies extra values that will be passed "
+    "to the SageMaker Endpoint invoke function."
+)
+
+MODEL_ARGS_SHORT_OPTION = "-m"
+MODEL_ARGS_LONG_OPTION = "--model-kwargs"
+MODEL_ARGS_HELP = (
+    "A JSON value that specifies extra values that will be passed to"
+    "the payload body of the invoke function. This can be useful to"
+    "pass model tuning parameters such as token count, temperature "
+    "etc., that affects the response generated by of a model."
```

Comment:
```suggestion
    "Extra values to be passed in the payload body of the invoke function. "
    "This can be used for model tuning parameters, such as token count and "
    "temperature, that affect a model's response."
```

Adds in spaces, removes "etc", which the AWS style guide discourages. Copy edits.